modules/ROOT/pages/introduction/use-exchange.adoc

```diff
@@ -0,0 +1,186 @@
+= Use Exchange to Discover Connectors, Templates, and Examples
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Use Anypoint Exchange (Exchange) to discover connectors, templates, and examples, and to incorporate them into your app. Exchange serves as a repository for connectors, templates, and examples provided both by MuleSoft and by other organizations. Searches in Exchange apply only to the Exchange organization you are viewing.
+
+Search in the https://www.mulesoft.com/exchange/[Public Exchange] to access MuleSoft's public assets. If you search in a private organization's Exchange, you need access rights to that organization.
```

Comment:
This was from Kuntal. Are you sure?

Reply:
It's correct that search in private Exchange organizations is access controlled, but confusing to say it this way to users, since they will never see a private Exchange organization but be blocked from searching in it. I omitted this sentence from my edited version, and I think that's the best way to explain Exchange searching as we introduce it in this page.